packages/service-library/src/data-providers/on-demand-page-scan-run-result-provider.spec.ts

```diff
@@ -133,12 +138,17 @@ describe(OnDemandPageScanRunResultProvider, () => {
                 .verifiable(Times.exactly(2));
 
             cosmosContainerClientMock
-                .setup(c => c.queryDocuments('select * from c where c.id in ("partition1id1", "partition1id2")', 'token1', 'bucket1'))
+                .setup(c =>
+                    c.queryDocuments(
+                        'select * from c where c.partitionKey = "bucket1" and c.id in ("partition1id1", "partition1id2")',
+                        'token1',
+                    ),
+                )
                 .returns(() => Promise.resolve({ item: call1Result } as CosmosOperationResponse<any[]>))
```

Comment:
did you run db.spec.ts tests? So, we don't have to make any changes to that file?